compiler/merge.cc

```diff
@@ -53,6 +53,71 @@ bool MaybeMergeSplitConcat(Graph* graph, Node* node) {
     return true;
 }
 
+bool MaybeMergePadConv(Graph* graph, Node* pad) {
+    if (pad->value() != 0.0 || pad->mode() != "constant") {
+        return false;
+    }
+
+    auto const& pads = pad->pads();
+
+    // range check
+    if (pads.size() < 4) {
+        return false;
+    }
+
+    // pads should apply to feature dims
```

Comment:
If you use the comment I suggested (i.e., "Padding for non-spatial dims cannot be merged."), I think we can remove this comment (and remove blank line in line 67)?